constraints.md

```diff
@@ -384,7 +384,7 @@ ALTER TABLE orders ADD FOREIGN KEY fk_user_id (user_id) REFERENCES users(id);
 
 ### Notes
 
-* TiDB supports foreign keys to avoid errors caused by this syntax when you migrate data from other databases to TiDB.
+* Before v6.6.0, TiDB supports foreign keys to avoid errors caused by this syntax when you migrate data from other databases to TiDB.
 
     However, TiDB does not perform constraint checking on foreign keys in DML statements. For example, even if there is no record with id=123 in the users table, the following transactions can be submitted successfully.
```

Comment:
Is line 389 applicable to v6.6.0? If not, can we remove the line 385~389 for v6.6.0 docs?

Reply:
Done https://github.com/pingcap/docs/pull/12275/commits/0f5109e2a71f49e25023f14a69042350c848b5f8